cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -145,6 +153,10 @@ func (cmd *command) deploy(start time.Time) error {
 		return err
 	}
 
+	if _, err := coredns.Patch(l.Desugar(), cmd.K8s.Static(), false, clusterInfo); err != nil {
```

Comment:
could you write a test for this?

Reply:
the same `coredns.Patch` is executed in `deploy command`.
The patching logic lives in its own package and it was introduced w/o tests.
I agree this needs tests, but I wouldn't add those as part of this PR.

Reply:
Just because a command is not tested is not exactly a reason for me to not test it in the future. Feel free to contribute tests as needed. If you feel like this should not be done here, then I would block this PR until the original Patching was tested. WDYT?

Reply:
how come the original contribution with patch command passed the review than ? 
Thats a mystery

Reply:
While I am not agreeing with this sentiment, I feel like I'm in the minority here so I will close the comment. I will say however that this way the CLI will never increase coverage. I am also gonna talk internally about more stricter standards in general for CLI so there won't be this argument anymore.